src/runner.js

```diff
@@ -4,13 +4,25 @@ const parser = require('./parser/parser.js');
 const results = require('./results.js');
 const DefaultVariableStorage = require('./default-variable-storage.js');
 const nodeTypes = require('./parser/nodes.js').types;
+const MessageFormat = require('messageformat');
 
 class Runner {
-  constructor() {
+  /**
+   * Creates an instance of Runner.
+   * @param {object} [options] Options for the runner
+   * @param {string} options.language Options for the language of the formater
+   * @memberof Runner
+   */
+  constructor(options) {
     this.yarnNodes = {};
     this.variables = new DefaultVariableStorage();
     this.functions = {};
     this.visited = {}; // Which nodes have been visited
+    this.options = {
+      language: 'en', // Handle messageFormat language change
+      ...options || {} // rewrite and insert if options contain data
+    };
+    this.messageFormater = new MessageFormat(this.options.language);
```

Comment:
if you want to make it possible for the user to set a custom formater, here you can say
`this.messageFormater = options.formater || new MessageFormat(this.options.language);`